packages/v4/src/content/design-guidelines/styles/typography/typography.css

```diff
@@ -8,6 +8,8 @@
 
 #usage-guidelines ~ h3.ws-title {
   margin-top: 24px;
+  font-size: 18px;
+  font-Weight: 400;
```

Comment:
I don't think you need to set `font-size` or `font-weight` here. Looking at this more, I think you can pass `className` to `<TitleLevel>` and pass the appropriate `pf-c-title pf-m-xl` classes to set the size to whatever you want.